apps/api/src/routes/v1/brawlhalla/get-servers.ts

```
@@ -0,0 +1,90 @@
+    Math.sin(dLat / 2) ** 2 +
+    Math.cos(toRad(lat1)) * Math.cos(toRad(lat2)) * Math.sin(dLon / 2) ** 2
+  const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a))
+  return R * c
+}
+
+export const getServers = Effect.fn("getServers")(function* () {
+  return {
+    data: servers,
+    meta: { timestamp: new Date() },
+  } satisfies typeof GetServersResponse.Type
+})
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)